extensions/exchanges/bitfinex/exchange.js

```diff
@@ -190,35 +184,31 @@ module.exports = function container (get, set, clear) {
         is_postonly
       }
       client.make_request('order/new', params, function (err, body) {
-        if (body.is_live === true) {
-          var order = {
-            id: body.id,
-            status: 'open',
-            price: opts.price,
-            size: opts.size,
-            post_only: !!opts.post_only,
-            created_at: new Date().getTime(),
-            filled_size: '0',
-            ordertype: opts.order_type
```

Comment:
was observing 'undefined' on maker sell. Buy was not affected. Don't know why...